src/main/java/database/service/JsonDatabaseService.java

```diff
@@ -154,6 +154,7 @@ public void removeAllRecordsFromTable(Class<? extends BaseEntity> entityClass) {
         Path databasePath = Path.of(getDatabasePath(entityClass));
         try {
             Files.writeString(databasePath, EMPTY_BRACKETS_TO_JSON);
+            entityIds.remove(entityClass.getName());
```

Comment:
1. Since we don't remove table here but only clear it, please do not remove counter, consider setting it up to initial value (-1).
2. Please remove counter when removing table in `deleteTable` method.